RELEASING.md

```diff
@@ -24,10 +24,33 @@ The following tasks are required before releasing:
 - Create a new release notes file
 - Update the `CHANGELOG.md` file
   
-  The `changelog.sh` script in `tools/scripts` will prepopulate the changelog but you must check and edit the file manually afterwards as required
+The `changelog.sh` script in `tools/scripts` will prepopulate the changelog but you must check and edit the file manually afterwards as required
+
+The `tools/scripts/updateversions.sh` script will update the version in all the `package.json` files.  Pass the new version as the first argument.
+
+```bash
+./tools/scripts/updateversions.sh 2.4.1
+```
 
 See the [Prepare 2.1.4 release](https://github.com/hyperledger/fabric-chaincode-node/pull/174) pull request for an example, although be careful to search for all versions in the codebase as they're easy to miss and things change!
 
+You can use this to help update the package.json. 
```

Comment:
Already mentioned the new updateversions.sh script above so this is not required